install_config/aggregate_logging.adoc

```diff
@@ -782,6 +820,54 @@ Please note that changes to the configmap might not appear until after redeployi
 the pods. Persisting these changes across deployments can be accomplished by setting
 `openshift_logging_es_allows_cluster_reader` to `true` in the inventory file.
 
+[[expose-elasticsearch-as-route]]
+*Expose Elasticsearch as a Route*
+
+By default, Elasticsearch deployed with OpenShift aggregated logging is not
+accessible from outside the logging cluster. This enables a route for external
+access to Elasticsearch for those tools that want to access its data.
+
+You now have direct access to Elasticsearch using only your OpenShift token and
+have the ability to provide the external Elasticsearch and Elasticsearch Ops
+hostnames when creating the server certificate (similar to Kibana). Ansible
+tasks now simplify route deployment.
+
+. To access Elasticsearch as a route, define the following variables:
+----
+openshift_logging_es_allow_external=True
+openshift_logging_es_hostname=elasticsearch.example.com
+----
+
+. Run the `openshift-logging.yml` Ansible playbook:
+----
+$ ansible-playbook [-i </path/to/inventory>] \
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/openshift-logging.yml
+----
+
+. To log in to Elasticsearch remotely, the request must contain two HTTP headers:
+----
+Authorization: Bearer $token
+----
+
+You must have access to the project in order to be able to access to the
+logs. For example:
+----
+$ oc login <user1>
+$ oc new-project <user1project>
+$ oc new-app <httpd-example>
+----
+
+You need to get the token of this ServiceAccount to be used in the request:
+----
+$ token=oc whoami -t
+----
+
+Using the token previously configured, you should be able access Elasticsearch
+through the exposed route:
+----
+$ curl -kv -H "Authorization: Bearer $token <URL>"
```

Comment:
-H "X-Proxy-Remote-User: $(oc whoami)" -H "X-Forwarded-For: 127.0.0.1" https://es.example.test/_cat/indices